### ISSUE TITLE
linux/netfilter/go.mod file outdated ?

### ISSUE THREAD

**Issue by benjaminforest** [mozilla-mobile/mozilla-vpn-client#2904]
I tried to build main, and it seems that the file `linux/netfilter/go.mod` do not have correct reference to nftables (the current one rise error about some repo not existing ) ? It seems linked to this commit : https://github.com/google/nftables/commit/edf9fe8cd04f49c758262e8b11654d8cb9ec53ed, that removed some references to koneu/natend repo that have been removed ?

```
require (
	github.com/frankban/quicktest v1.13.0 // indirect
	github.com/google/nftables
...
```
I dont feel like doing a PR, since I don't really know which version of nftables should be used.
All the best,

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-1854)

**Comment by oskirby** [comment]
Updating to the latest version of nftables appears to require us to migrate to golang version 1.16, but unfortunately this would require breaking the packaging on Ubuntu/bionic, which only supports golang 1.13. I will see if I can find a workaround in the meantime.

**Comment by umeyatakuma** [comment]
What's the prospect of  this bug? I'm hitting the same issue. Thanks,

**Comment by oskirby** [comment]
This should be fixed by #3898
